src/background/tab-manager.ts

```diff
@@ -60,7 +73,7 @@ export default class TabManager {
 
                 // Using custom response due to Chrome and Firefox incompatibility
                 // Sometimes fetch error behaves like synchronous and sends `undefined`
-                const sendResponse = (response) => chrome.tabs.sendMessage(sender.tab.id, {type: 'fetch-response', id, ...response});
+                const sendResponse = (response) => chrome.tabs.sendMessage(sender.tab == undefined ? sender.id as any : sender.tab.id, {type: 'fetch-response', id, ...response});
```

Comment:
`== null` is preferred. Why do you cast` sender.id` to `any`, what is the actual value, is it a number? Maybe it's a type definition bug.

Reply:
It's a string, but that doesn't get accepted as type and also it's gives a error in the console for some reason